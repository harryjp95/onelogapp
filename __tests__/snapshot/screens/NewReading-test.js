import 'react-native'
import React from 'react'
import renderer from 'react-test-renderer'

import NewReadingScreen from '../../../src/Components/Screens/Settings'

it('renders correctly', () => {
  const newReadingScreen = renderer.create(<NewReadingScreen />)

  expect(newReadingScreen).toMatchSnapshot()
})
