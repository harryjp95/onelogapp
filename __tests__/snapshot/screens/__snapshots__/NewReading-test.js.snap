// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#ebebeb",
      "flexDirection": "column",
      "height": 1214,
      "justifyContent": "flex-start",
      "width": 750,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "blue",
        "flexDirection": "row",
      }
    }
  >
    <View>
      <BVLinearGradient
        colors={
          Array [
            4293651435,
            4286611584,
            4293651435,
          ]
        }
        endPoint={
          Object {
            "x": 0.5,
            "y": 0.5,
          }
        }
        locations={null}
        startPoint={
          Object {
            "x": 0,
            "y": 1,
          }
        }
        style={
          Object {
            "alignItems": "center",
            "height": 0.5,
            "justifyContent": "center",
            "width": "100%",
          }
        }
      />
    </View>
    <View>
      <Text>
        HELLO
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "backgroundColor": "blue",
        "flexDirection": "row",
      }
    }
  >
    <View>
      <BVLinearGradient
        colors={
          Array [
            4293651435,
            4286611584,
            4293651435,
          ]
        }
        endPoint={
          Object {
            "x": 0,
            "y": 0,
          }
        }
        locations={null}
        startPoint={
          Object {
            "x": 0,
            "y": 1,
          }
        }
        style={
          Object {
            "alignItems": "center",
            "height": 0.5,
            "justifyContent": "center",
            "width": "100%",
          }
        }
      />
    </View>
    <View />
  </View>
</View>
`;
